### DISCUSSION TITLE
Locale IT Feedback

### DISCUSSION THREAD

**Discussion by rbernadev** [ueman/feedback#335] [Q&A]
`class ItFeedbackLocalizations implements FeedbackLocalizations {

  const ItFeedbackLocalizations();

  @override
  String get submitButtonText => 'Invia Segnalazione';

  @override
  String get feedbackDescriptionText => 'Cosa non funziona come dovrebbe?';

  @override
  String get draw => 'Disegna';

  @override
  String get navigate => 'Naviga';
}`

**Comment by ueman** [comment]
What is this supposed to mean?

**Comment by rbernadev** [comment]
Is italian translation of feedback plugin

Regards
Riccardo

Il sab 16 nov 2024, 19:32 Jonas Uekötter ***@***.***> ha
scritto:
